docs/security.html

```diff
@@ -2089,6 +2089,144 @@ <h5 class="anchor-heading"><a id="operations_resources_and_protocols" class="anc
             <td>Topic</td>
             <td></td>
         </tr>
+        <tr>
+            <td>DESCRIBE_CLIENT_QUOTAS (48)</td>
+            <td>DescribeConfigs</td>
+            <td>Cluster</td>
+            <td></td>
+        </tr>
+        <tr>
+            <td>ALTER_CLIENT_QUOTAS (49)</td>
+            <td>AlterConfigs</td>
+            <td>Cluster</td>
+            <td></td>
+        </tr>
+        <tr>
+            <td>DESCRIBE_USER_SCRAM_CREDENTIALS (50)</td>
+            <td>Describe</td>
+            <td>Cluster</td>
+            <td></td>
+        </tr>
+        <tr>
+            <td>ALTER_USER_SCRAM_CREDENTIALS (51)</td>
+            <td>Alter</td>
+            <td>Cluster</td>
+            <td></td>
+        </tr>
+        <tr>
+            <td>VOTE (52)</td>
+            <td>ClusterAction</td>
+            <td>Cluster</td>
+            <td></td>
+        </tr>
+        <tr>
+            <td>BEGIN_QUORUM_EPOCH (53)</td>
+            <td>ClusterAction</td>
+            <td>Cluster</td>
+            <td></td>
+        </tr>
+        <tr>
+            <td>END_QUORUM_EPOCH (54)</td>
+            <td>ClusterAction</td>
+            <td>Cluster</td>
+            <td></td>
+        </tr>
+        <tr>
+            <td>DESCRIBE_QUORUM (55)</td>
+            <td>ClusterAction</td>
+            <td>Cluster</td>
+            <td></td>
+        </tr>
+        <tr>
+            <td>DESCRIBE_QUORUM (55)</td>
+            <td>Alter</td>
+            <td>Cluster</td>
+            <td></td>
+        </tr>
```

Comment:
Duplicated entry. And it should be `Describe` `Cluster`, right?

Reply:
Re duplication, I commented below. And yes, it should have been Describe. Thanks for spotting that :)